packages/react/src/box/styles.ts

```diff
@@ -269,6 +269,12 @@ type LayoutProps = Partial<{
 	alignItems: ResponsiveProp<
 		'stretch' | 'flex-start' | 'flex-end' | 'center' | 'baseline'
 	>;
+	/** Sets the CSS align-self property.
+	 * @see https://developer.mozilla.org/en-US/docs/Web/CSS/align-self
+	 */
+	alignSelf: ResponsiveProp<
+		'stretch' | 'start' | 'end' | 'center' | 'baseline'
```

Comment:
We're using `start` here, but `flex-start` in `alignItems`. Not sure why we limit the ones folks can use, but if we are limiting, then they should be equal (and probably include both starts and ends)

Reply:
Yes, good spot. I started looking into this and forgot to come back. I don't think we should exclude any options, really. Another reason to use the `CSSObject` type from emotion.

What do you think about this for both:  `'stretch' | 'start' | 'flex-start' | 'end' | 'flex-end' | 'center' | 'baseline'`

Reply:
Yep, works for me. Will need a documenting in the changeset also

Reply:
- [ ] `'stretch' | 'start' | 'flex-start' | 'end' | 'flex-end' | 'center' | 'baseline'`
- [ ] changeset